src/Main.elm

```diff
@@ -420,12 +419,12 @@ type FormFieldMsg
     | OnVisibilityConditionDuplicate Int
 
 
-otherQuestionTitles : Array FormField -> Int -> List String
+otherQuestionTitles : Array FormField -> Int -> List { label : String, name : Maybe String }
 otherQuestionTitles formFields currentIndex =
     Array.toList formFields
         |> List.indexedMap (\i f -> ( i, f ))
         |> List.filter (\( i, _ ) -> i /= currentIndex)
-        |> List.map (\( _, f ) -> f.label)
+        |> List.map (\( _, f ) -> { label = f.label, name = f.name })
```

Comment:
fixed to respect `name` attribute of "other question" in our logic target dropdown selection